scss/cayman.scss

```diff
@@ -1,6 +1,21 @@
+// Breakpoints
 $large-breakpoint: 64em;
 $medium-breakpoint: 42em;
 
+// Sass variables
+$brand-color: #159957;
+$secondary-brand-color: #155799;
+$light-color: #FFFFFF;
```

Comment:
This seems to be mostly used for the various `rgba()` colors, so let's not use a hex value. For the main heading, we could probably define `$header-heading-color` and then maybe make the other variables match up (e.g. `$header-bg-color` and `$header-bg-color-secondary`).